packages/site/src/data/react/libraries.ts

```diff
@@ -1026,4 +1026,16 @@ export const libraries: Library[] = [
 		tags: [LibraryTag.ROUTING],
 		language: 'TypeScript',
 	},
+	{
+		name: 'Ignite UI for React',
+		author: 'Infragistics',
+		repo: 'https://github.com/IgniteUI/igniteui-react',
+		package: 'https://www.npmjs.com/package/igniteui-react',
+		href: 'https://www.infragistics.com/products/ignite-ui-react',
+		image: 'https://static.infragistics.com/marketing/Website/products/ignite-ui-landing/ignite-ui-logo.svg',
```

Comment:
Let's make the same change as the Angular site
```suggestion
		image: 'https://avatars.githubusercontent.com/u/5366066?s=200&v=4',
```